### PR TITLE
(feat) Remove redundant translation configuration settings

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR removes some redundant translation configuration from the `Start Visit` component that affects how the button label gets translated in the UI. For such scenarios, translation configuration should leverage the [translation overrides](https://o3-docs.openmrs.org/docs/configureO3YourWay/translation-in-o3#translation-overrides-specific-to-an-individual-o3-module) mechanism instead.

## Related Issue
*None*

## Other
*None*

### CODE REVIEW

**Review by vasharma05 — APPROVED** [review]

**Comment by denniskigen** [comment]
Thanks, @donaldkibet. Don't forget to update your config appropriately.

**Comment by jayasanka-sack** [comment]
@donaldkibet It seems the server is throwing `503`s when performing some actions. Could you please look into it and let us know whether it is something related to your changes or a bug on the product itself? 

Instructions on how to check the test report can be found here: https://github.com/openmrs/openmrs-esm-patient-chart/tree/main/e2e#open-reports-from-github-actions--bamboo



image

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
@donaldkibet It seems the server is throwing `503`s when performing some actions. Could you please look into it and let us know whether it is something related to your changes or a bug on the product itself? 

Instructions on how to check the test report can be found here: https://github.com/openmrs/openmrs-esm-patient-chart/tree/main/e2e#open-reports-from-github-actions--bamboo



image

**Comment by donaldkibet** [comment]
@jayasanka-sack looking into the issue, where the e2e test are failing is un-related to the changes that I have made.

**Review by CynthiaKamau — APPROVED** [review]

File: packages/esm-patient-chart-app/src/visit/start-visit-button.component.tsx [hunk 1]
```diff
@@ -2,15 +2,10 @@ import React, { useCallback } from 'react';
...
-import { navigate, useConfig } from '@openmrs/esm-framework';
-
-interface StartVisitButtonProps {
-  patientUuid: string;
-}
+import { navigate } from '@openmrs/esm-framework';
 
 const StartVisitButton = ({ patientUuid }) => {
   const { t } = useTranslation();
-  const { startVisitLabel } = useConfig();
```
Comment (vasharma05) [inline packages/esm-patient-chart-app/src/visit/start-visit-button.component.tsx:13]: Can we also remove the `startVisitLabel` point from config as well?
Thanks!

**Pull request merged**